### ISSUE TITLE
yt-dlp downloads (e.g. YouTube) audio and video stream files, muxes the intermediates, but then deletes the output file

### ISSUE THREAD

**Issue by ChristopherW** [yt-dlp/yt-dlp#6076]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2023.01.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

When downloading any video, but taking an example from today with a YouTube video, the intermediate video and audio files for specific quality level are initially downloaded and saved successfully. FFmpeg muxes them (either to the default or a specified container), but immediately after that when the script does the intermediate clean-up, the final output files appear to be also considered intermediate or temporary and are always deleted. In order to keep the final muxed file I must specify the `-k` switch, which also leaves the separate audio and video files.

What appears to be happening is that the audio and video files are downloaded then FFmpeg muxes them to the output file with the name I specify. A further FFmpeg remux then runs, specifies no video, and copies the audio stream to an m4a file, which is then deleted along with all the other files.

I've tried with an independently sourced win64, the latest win64 5.1, the win64 `N-109649-gab8cde6efa-20230122` build and a slightly older win32 build from the yt-dlp/FFmpeg-Builds repo.

I can see the files created and growing in size both during download and remux, but all are deleted silently when the task completes. No obvious errors are thrown.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
> yt-dlp -vU -f 137+140 --merge-output-format "mp4" https://www.youtube.com/watch?v=P_dxzwQ5R04 -o "C:\apps\av\ytdlp\supernaturalsongs.mp4"

C:\Apps\AV\ytdlp>yt-dlp -vU -f 137+140 --merge-output-format "mp4" https://www.youtube.com/watch?v=P_dxzwQ5R04 -o "C:\temp\supernaturalsongs.mp4"
[debug] Command-line config: ['-vU', '-f', '137+140', '--merge-output-format', 'mp4', 'https://www.youtube.com/watch?v=P_dxzwQ5R04', '-o', 'C:\\temp\\supernaturalsongs.mp4']
[debug] User config "C:\Users\Christopher\AppData\Roaming\yt-dlp\config": ['--cookies-from-browser', 'chrome', '-v', '-x', '--add-metadata']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2023.01.06 [6becd25] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19043-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg N-109649-gab8cde6efa-20230122 (setts), ffprobe N-109649-gab8cde6efa-20230122
[debug] Optional libraries: Cryptodome-3.16.0, brotli-1.0.9, certifi-2022.12.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[Cookies] Extracting cookies from chrome
[debug] Extracting cookies from: "C:\Users\Christopher\AppData\Local\Google\Chrome\User Data\Default\Network\Cookies"
[debug] Found local state file at "C:\Users\Christopher\AppData\Local\Google\Chrome\User Data\Local State"
[Cookies] Extracted 3056 cookies from chrome
[debug] cookie version breakdown: {'v10': 3056, 'other': 0, 'unencrypted': 0}
[debug] Proxy map: {}
[debug] Loaded 1760 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2023.01.06, Current version: 2023.01.06
yt-dlp is up to date (2023.01.06)
[youtube] Extracting URL: https://www.youtube.com/watch?v=P_dxzwQ5R04
[youtube] P_dxzwQ5R04: Downloading webpage
[debug] [youtube] Extracted SAPISID cookie
[youtube] P_dxzwQ5R04: Downloading android player API JSON
[youtube] P_dxzwQ5R04: Downloading MPD manifest
[youtube] P_dxzwQ5R04: Downloading MPD manifest
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] P_dxzwQ5R04: Downloading 1 format(s): 137+140
[debug] Invoking http downloader on "https://rr4---sn-aigl6nek.googlevideo.com/videoplayback?expire=1674424461&ei=LVzNY7D5A8jCWcCPkIgC&ip=2001%3A4d48%3Aad5d%3A8400%3A9cbe%3A5a6e%3Afef6%3A3540&id=o-AKICT1vLyWYPMicZTGNqJTJwl60M57Amzwo1vxeHdp61&itag=137&source=youtube&requiressl=yes&mh=76&mm=31%2C29&mn=sn-aigl6nek%2Csn-aigzrn7s&ms=au%2Crdu&mv=m&mvi=4&pl=32&ctier=A&pfa=5&initcwndbps=998750&hightc=yes&spc=zIddbDGEemm_IYx1BZuJO2cAyPcRsoLLwQfiemBy17A7&vprv=1&svpuc=1&mime=video%2Fmp4&gir=yes&clen=763524214&otfp=1&dur=1779.000&lmt=1669714966523176&mt=1674402330&fvip=1&keepalive=yes&fexp=24007246&c=ANDROID&txp=6216224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cctier%2Cpfa%2Chightc%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cotfp%2Cdur%2Clmt&sig=AOq0QJ8wRQIgW95KTR0171mkg6sMaFJggq3u09-ldUuWoRWcccVjkOkCIQDnCLsWqiee5qOzUELMsCI8Q8fasHy5nPricqwDusww2A%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgfsfOmDzj0s6yDDmYEQJ1xTMmIzI_AsrJi8C3xLQu7Z4CIDeRkDWysLFt_ApI0kjhCFFCmfTY4GVasZWQ-9dmZN1o"
[debug] File locking is not supported. Proceeding without locking
[download] Destination: C:\temp\supernaturalsongs.f137.mp4
[download] 100% of  728.15MiB in 00:00:22 at 32.25MiB/s
[debug] Invoking http downloader on "https://rr4---sn-aigl6nek.googlevideo.com/videoplayback?expire=1674424461&ei=LVzNY7D5A8jCWcCPkIgC&ip=2001%3A4d48%3Aad5d%3A8400%3A9cbe%3A5a6e%3Afef6%3A3540&id=o-AKICT1vLyWYPMicZTGNqJTJwl60M57Amzwo1vxeHdp61&itag=140&source=youtube&requiressl=yes&mh=76&mm=31%2C29&mn=sn-aigl6nek%2Csn-aigzrn7s&ms=au%2Crdu&mv=m&mvi=4&pl=32&ctier=A&pfa=5&initcwndbps=998750&hightc=yes&spc=zIddbDGEemm_IYx1BZuJO2cAyPcRsoLLwQfiemBy17A7&vprv=1&svpuc=1&mime=audio%2Fmp4&gir=yes&clen=28792963&otfp=1&dur=1779.066&lmt=1669714717376169&mt=1674402330&fvip=1&keepalive=yes&fexp=24007246&c=ANDROID&txp=6211224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cctier%2Cpfa%2Chightc%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cotfp%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAPRLcE-bWa5IkoFtVlmeM3rOprYqO3WQL0TON3qyY3tZAiEAlE2ASEwrgcD1LdO4j4-dpn4EoufXiuZnIS456B8weVY%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgfsfOmDzj0s6yDDmYEQJ1xTMmIzI_AsrJi8C3xLQu7Z4CIDeRkDWysLFt_ApI0kjhCFFCmfTY4GVasZWQ-9dmZN1o"
[download] Destination: C:\temp\supernaturalsongs.f140.m4a
[download] 100% of   27.46MiB in 00:00:00 at 33.63MiB/s
[Merger] Merging formats into "C:\temp\supernaturalsongs.mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:C:\temp\supernaturalsongs.f137.mp4" -i "file:C:\temp\supernaturalsongs.f140.m4a" -c copy -map "0:v:0" -map "1:a:0" -movflags "+faststart" "file:C:\temp\supernaturalsongs.temp.mp4"
Deleting original file C:\temp\supernaturalsongs.f140.m4a (pass -k to keep)
Deleting original file C:\temp\supernaturalsongs.f137.mp4 (pass -k to keep)
[debug] ffmpeg command line: ffprobe -show_streams "file:C:\temp\supernaturalsongs.mp4"
[ExtractAudio] Destination: C:\temp\supernaturalsongs.m4a
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:C:\temp\supernaturalsongs.mp4" -vn -acodec copy "-bsf:a" aac_adtstoasc -movflags "+faststart" "file:C:\temp\supernaturalsongs.m4a"
Deleting original file C:\temp\supernaturalsongs.mp4 (pass -k to keep)
[Metadata] Adding metadata to "C:\temp\supernaturalsongs.m4a"
[debug] ffmpeg command line: ffmpeg -y -loglevel "repeat+info" -i "file:C:\temp\supernaturalsongs.m4a" -map 0 -dn -ignore_unknown -vn -acodec copy -write_id3v1 1 -metadata "title=Supernatural Songs - Tavener // RNS & Dame Sarah Connolly (Inspired by Lindisfarne Gospels)" -metadata "date=20221201" -metadata "description=Follow the journey of the Lindisfarne Gospels in this special concert film with Royal Northern Sinfonia, transporting you through the stunning North East landscape and into the iconic illustrated pages of the Gospels.

As part of the North East-wide celebrations marking the return of the Lindisfarne Gospels to the region in a major exhibition at Newcastle’s Laing Gallery, we presented a special concert featuring the rarely heard work, John Tavener’s Supernatural Songs. Dame Sarah Connolly, a renowned champion of Tavener’s music, was our soloist, alongside Royal Northern Sinfonia, conducted by James Weeks.

To accompany the performance, we commissioned a brand-new film work by Newcastle-based studio, NOVAK, taking inspiration from the Lindisfarne Gospels and Tavener’s deeply spiritual and meditative music.

Performed live at Sage Gateshead on Friday 23 September 2022.

Available to watch until February 2023.

Supernatural Songs - John Tavener
James Weeks: conductor
Dame Sarah Connolly:  mezzo-soprano
Royal Northern Sinfonia

Creative Team
Concert Film Director: Mary Crothers
Sound Mix: Richard Halling
Stream provided by 8 Live

Direction, Editing and Event Playback: NOVAK
Drone Operator: James Winnifrith
---

Roots in the region, renowned around the world.

Royal Northern Sinfonia, orchestra of Sage Gateshead, is the UK’s only full-time chamber orchestra. Founded in 1958, RNS has built a worldwide reputation for the North East through the quality of its music-making and the immediacy of the connections the musicians make with audiences.

Subscribe: https://bit.ly/3I0YQxZ
Like: https://www.facebook.com/royalnorthernsinfonia
Follow: https://twitter.com/RNSinfonia
Discover: https://www.instagram.com/royal_northern_sinfonia/
What’s on? http://www.sagegateshead.com/royalnorthernsinfonia

Welcome to Royal Northern Sinfonia's YouTube channel. Subscribe to the Royal Northern Sinfonia YouTube Channel for regular performance films recorded live at Sage Gateshead, interviews and behind the scenes features and special short film commissions." -metadata "synopsis=Follow the journey of the Lindisfarne Gospels in this special concert film with Royal Northern Sinfonia, transporting you through the stunning North East landscape and into the iconic illustrated pages of the Gospels.

As part of the North East-wide celebrations marking the return of the Lindisfarne Gospels to the region in a major exhibition at Newcastle’s Laing Gallery, we presented a special concert featuring the rarely heard work, John Tavener’s Supernatural Songs. Dame Sarah Connolly, a renowned champion of Tavener’s music, was our soloist, alongside Royal Northern Sinfonia, conducted by James Weeks.

To accompany the performance, we commissioned a brand-new film work by Newcastle-based studio, NOVAK, taking inspiration from the Lindisfarne Gospels and Tavener’s deeply spiritual and meditative music.

Performed live at Sage Gateshead on Friday 23 September 2022.

Available to watch until February 2023.

Supernatural Songs - John Tavener
James Weeks: conductor
Dame Sarah Connolly:  mezzo-soprano
Royal Northern Sinfonia

Creative Team
Concert Film Director: Mary Crothers
Sound Mix: Richard Halling
Stream provided by 8 Live

Direction, Editing and Event Playback: NOVAK
Drone Operator: James Winnifrith
---

Roots in the region, renowned around the world.

Royal Northern Sinfonia, orchestra of Sage Gateshead, is the UK’s only full-time chamber orchestra. Founded in 1958, RNS has built a worldwide reputation for the North East through the quality of its music-making and the immediacy of the connections the musicians make with audiences.

Subscribe: https://bit.ly/3I0YQxZ
Like: https://www.facebook.com/royalnorthernsinfonia
Follow: https://twitter.com/RNSinfonia
Discover: https://www.instagram.com/royal_northern_sinfonia/
What’s on? http://www.sagegateshead.com/royalnorthernsinfonia

Welcome to Royal Northern Sinfonia's YouTube channel. Subscribe to the Royal Northern Sinfonia YouTube Channel for regular performance films recorded live at Sage Gateshead, interviews and behind the scenes features and special short film commissions." -metadata "purl=https://www.youtube.com/watch?v=P_dxzwQ5R04" -metadata "comment=https://www.youtube.com/watch?v=P_dxzwQ5R04" -metadata "artist=Royal Northern Sinfonia" -movflags "+faststart" "file:C:\temp\supernaturalsongs.temp.m4a"
```

**Comment by Grub4K** [comment]
I don't quite understand the problem, yt-dlp does exactly what asked: download video (`C:\temp\supernaturalsongs.f137.mp4`) and audio (`C:\temp\supernaturalsongs.f140.m4a`) and merge them (`-f 137+140`) into one `mp4` (`C:\temp\supernaturalsongs.mp4`, `--merge-output-format "mp4"`), deleting the temp files in the process. Then it extracts the audio (`-x`) to `C:\temp\supernaturalsongs.m4a`, deleting that temp file (`C:\temp\supernaturalsongs.mp4`) and adds metadata (`--add-metadata`) in place. This is reflected in the log as well.

What is the problem here?
Do you want to use the intermediate file directly? (`C:\temp\supernaturalsongs.f140.m4a`)
What are your desired output files with what content inside?

**Comment by dirkf** [comment]
Or, in other words, you probably don't want `-x` in the config file.

**Comment by ChristopherW** [comment]
I couldn't understand why yt-dlp was merging then deleting the muxed file after extracting the audio. 

Ready for a chuckle? I had completely set-and-forgotten the config file. For some reason I put it in my Roaming appdata instead of the portable in the install directory named more memorably, no idea why. It was to specify the cookies-from-browser and add-metadata switches, but I added in `-x` for whatever reason when I made the file 14 months ago. I could only remember specifying YouTube credentials in a `netrc`. 

Summary: ID10T error confirmed, failure to parse the exploded command-line config parameters, embarrassing ticket logged. Resolution: user will be sent for reeducation and reminded to document his config choices somewhere.
Thanks for your patience and leniency ;-)
